docs/pages/get-started/react.mdx

```diff
@@ -134,26 +140,28 @@ We can re-export this from `liveblocks.config.js`, and because we’re using
 suspense in this example, we’ll export all our hooks from the `suspense`
 property.
 
-```jsx file="liveblocks.config.js" highlight="11"
+```ts file="liveblocks.config.ts" highlight="11"
 import { createClient } from "@liveblocks/client";
 import { createRoomContext } from "@liveblocks/react";
+import { Presence, UserMeta, Storage } from "./types";
 
 const client = createClient({
   publicApiKey: "{{PUBLIC_KEY}}",
 });
 
+
 export const {
   suspense: {
     RoomProvider,
     useOthers, // 👈
   },
-} = createRoomContext(client);
+} = createRoomContext< Storage, UserMeta>(client);
```

Comment:
These type params need to be in a specific order. All of them are optional, but the order is always fixed. Each of these annotations are valid, but no others:

```ts
createRoomContext<Presence>(client);
createRoomContext<Presence, Storage>(client);
createRoomContext<Presence, Storage, UserMeta>(client);
createRoomContext<Presence, Storage, UserMeta, RoomEvent>(client);
```

Reply:
Exact, as we just start to introduce `useOthers`, we need to simply define a new `Presence` type in this code snippet and let the developer know that explain that a bit bellow.